codalab/codalab/context_processors.py

```diff
@@ -22,4 +27,5 @@ def common_settings(request):
         'is_dev': codalab_settings.IS_DEV,
         'USE_AWS': codalab_settings.USE_AWS,
         'CODALAB_SITE_DOMAIN': codalab_settings.CODALAB_SITE_DOMAIN,
+        'USE_MAILCHIMP': use_mailchimp,
```

Comment:
instead of the temp var `use_mailchimp` can do..

```python
'USE_MAILCHIMP': bool(settings.MAILCHIMP_API_KEY)
```

(ha! makes me remember 'vool' from like 11 years ago :D)

Reply:
Added vool() to latest push, Such a clean way to do this!